internal/pkg/build/assemblers/assembler_sif.go

```diff
@@ -146,7 +146,7 @@ func (a *SIFAssembler) Assemble(b *types.Bundle, path string) (err error) {
 	os.Remove(squashfsPath)
 	defer os.Remove(squashfsPath)
 
-	args := []string{b.Rootfs(), squashfsPath, "-noappend"}
+	args := []string{b.Rootfs(), squashfsPath, "-noappend", "-comp", "gzip"}
```

Comment:
There are versions of mksquashfs on distro's we support that do not have the `-comp` flag